lib/shared/models/selection-state.js

```diff
@@ -308,33 +344,54 @@ exports.removeImage = () => {
  */
 exports.clear = (options = {}) => {
   if (!options.preserveImage) {
-    store.dispatch({
-      type: store.Actions.REMOVE_IMAGE
-    })
+    exports.removeImage()
   }
 
-  store.dispatch({
-    type: store.Actions.REMOVE_DRIVE
-  })
+  _.map(exports.getSelectedDevices(), exports.removeDrive)
```

Comment:
Would it be worth having a `.removeAllDrives()` function?

Reply:
I recall using `selectionState.clear({ preserveImage: true })` as a `.removeAllDrives()` so good question, I think we should add the function and replace those `.clear()`s for clarity.